### ISSUE TITLE
Exact Training Details to Reproduce Tiny Model Results 

### ISSUE THREAD

**Issue by achen46** [google-research/maxvit#8]
@vztu I wonder if you can please share the exact training details to reproduce the MaxViT-T performance (83.62%). By this, I wonder about the total batch size, number of GPU/TPUs, nodes, learning rate and other important considerations.

**Comment by vztu** [comment]
Exactly shown below. We trained on [TPUv2-128](https://cloud.google.com/tpu/docs/regions-zones#overview) using batch-size 4096. We used a training script very similar to [efficientnet-v1](https://github.com/tensorflow/tpu/tree/master/models/official/efficientnet).

Our performance has been reproduced in the timm lib as well: https://github.com/rwightman/pytorch-image-models#oct-10-2022

![image](https://user-images.githubusercontent.com/43280278/197876697-555b46f4-a37c-436f-b887-5ffd3fbda94e.png)

**Comment by achen46** [comment]
Thanks ! for ImageNet-1K, what does pre-training mean ? usually people just train it once as it is a small dataset. Did you fine-tune again for 30 epochs ? then how's your results comparable to Swin, ConvNext, etc. which are not pre-trained then fine-tuned on ImageNet-1K ? 

Another point, I just did a throughput comparison between MaxViT-T, Swin-T and ConvNext-T, and MaxViT-T is about 1.5 and 2 times slower than Swin-T and ConvNext-T. So it is really not an efficient network.

**Comment by achen46** [comment]
Regarding the "huger performance gain", in my opinion, the results are misleading. MaxViT models often has substantially more number of parameters than the counterparts. For example, base models usually have 90M parameter where as MaxViT-B has 120M parameters. Same with the MaxViT-T. If you match the number of parameters and also the number of activations, MaxViT performance does not have "huger performance gain". 

That being said, it is better to compare apples to oranges to give the readers better understanding of the performance.
